ibvl/management/tools.py

```diff
@@ -0,0 +1,1306 @@
+                        )
+                        continue
+                    if self.batch:
+                        if not self.check_existing(row):
+                            success, obj = self.created_row_object(row)
+                            if success:
+                                bulk_create.append(obj)
+                            else:
+                                errors.append(obj)
+                                if self.failfast:
+                                    return errors, warnings
+                        elif self.update_existing:
+                            success, obj = self.update(row)
+                            if not success:
+                                errors.append(obj)
+                                if self.failfast:
+                                    return errors, warnings
+                    else:
+                        success, obj = self.update_or_create(row)
+                        if not success:
+                            errors.append(obj)
+                            if self.failfast:
+                                return errors, warnings
+
+                    if len(bulk_create) >= self.batch_size:
+                        try:
+                            self.model.objects.bulk_create(bulk_create)
+                            bulk_create = []
+                        except Exception as e:
+                            msg = (
+                                f"error in bulk create of {self.object_name_plural} "
+                                f"after input line {self.reader.line_num}: {e}"
+                            )
+                            errors.append(msg)
+                            if self.failfast:
+                                return errors, warnings
+                # Run out of input rows, tidy up outstanding create/updates
+                if len(bulk_create):
+                    try:
+                        self.model.objects.bulk_create(bulk_create)
+                        bulk_create = []
+                    except Exception as e:
+                        msg = (
+                            f"error in bulk create of {self.object_name_plural} "
+                            f"after end of input {self.reader.line_num}: {e}"
+                        )
+                        errors.append(msg)
+                        if self.failfast:
+                            return errors, warnings
+            except csv.Error as e:
+                errors.append(f"error reading line {self.reader.line_num}: {e}")
+                if self.failfast:
+                    return errors, warnings
+        return errors, warnings
+
+
+class SeverityImporter(Importer):
+    """Importer class to locate the severities data file and load severities from it"""
+
+    model = ibvlmodels.Severity
+    path_component = "severities"
+
+    def get_input_path(self):
+        return self.path / f"{self.path_component}.tsv"
```

Comment:
Please verify that Ben's data will include a TSV file for severities.  If so, make sure it's committed in fixtures so that dev bootstrapping is seamless

Reply:
Yep, at the moment I created one based on the previously-hardcoded data. I'll add that in.

Reply:
I am thinking the fixtures should represent the same structure as Ben's data. So, if Ben's data does not have severities, IMO it should be committed somewhere NOT in fixtures.

Reply:
Hi @nwp90, if you can please set `data/fixtures/` to be the default import path and commit `severities.tsv` then I would like to have this PR closed and merged